primers/artificial-intelligence/README.md

```diff
@@ -1,91 +1,44 @@
-﻿# Artificial Intelligence Primer: Introduction to the Subject
-
-## [Introduction to Artificial Intelligence](#_toc140607934)
-[Definition and Overview](#_toc140607935)\
-[Importance and Applications](#_toc140607936)\
-[Ethical Considerations](#_toc140607937)\
-[Accessibility](#_toc140607939)\
-[Security](#_toc140607940)\
-[Emerging Current Trends](#_toc140607941)\
-[What comes next?](#_toc140607942)
-
-## [A Brief History of Artificial Intelligence](#_toc140607943)
-[Early Developments](#_toc140607944)\
-[Key Milestones](#_toc140607945)\
-[Current State & Future Directions](#_toc140607946)
-- [Machine Learning](#_toc140607947)
-- [Natural Language Processing](#_toc140607948)
-- [Computer Vision](#_toc140607949)
-- [Robotics and Autonomous Systems](#_toc140607950)
-- [Explainable AI and Ethical Considerations](#_toc140607951)
-- [Emerging Trends](#_toc140607952)
-- [Future Directions](#_toc140607953)
-
-## [Tools for Artificial Intelligence Development](#_toc140607954)
-[Programming Languages](#_toc140607955)
-- [Python](#_toc140607956)
-- [R](#_toc140607957)
-- [Julia](#_toc140607958)
-- [Java](#_toc140607959)
-- [C++](#_toc140607960)
-- [MATLAB](#_toc140607961)
-
-[Frameworks & Libraries](#_toc140607962)
-- [TensorFlow](#_toc140607963)
-- [PyTorch](#_toc140607964)
-- [scikit-learn](#_toc140607965)
-- [Keras](#_toc140607966)
-- [MXNet](#_toc140607967)
-
-[Integrated Development Environments (IDEs)](#_toc140607968)
-- [PyCharm](#_toc140607969)
-- [Jupyter Notebooks](#_toc140607970)
-- [Visual Studio Code](#_toc140607971)
-- [Spyder](#_toc140607972)
-
-[Cloud Platforms and Services](#_toc140607973)
-- [Amazon Web Services (AWS)](#_toc140607974)
-- [Google Cloud Platform (GCP)](#_toc140607975)
-- [Microsoft Azure](#_toc140607976)
-- [IBM Watson](#_toc140607977)
-- [Oracle Cloud Infrastructure (OCI)](#_toc140607978)
-
-## <a name="_toc140607934"></a>Introduction to Artificial Intelligence
+﻿---
+id: artificial-intelligence
+title: "Artificial Intelligence Primer"
```

Comment:
do we need "Primer" in the name? other two primers don't have it

Reply:
*Spatial computing doesn't have it, DLT does, so need to decide whether to add it for concistency

Reply:
I'd say to drop it